tests/library/chromium/chromium.spec.ts

```diff
@@ -390,6 +393,7 @@ test('should emit page-level request event for respondWith when interception ena
   expect(pageReq.url()).toMatch(/fetchdummy\/foo$/);
   expect(pageReq.serviceWorker()).toBe(null);
   expect((await pageReq.response()).fromServiceWorker()).toBe(true);
+  expect(markFailureIfPageRoutesARequestAlreadyHandledByServiceWorker).toBe(false);
```

Comment:
Let's also test that we get `response` and `requestfinished` events.

Reply:
updating. also adding requestfailed case, too.